### PR TITLE
PCM-7771: Passes FL param to rs/cases search and export to csv

### PR DESCRIPTION
**Pull request by praniredhat**
Please review @ascension @vrathee @anujsi

### CODE REVIEW

**Comment by praniredhat** [comment]
pcm-ascension-commons : `https://github.com/redhataccess/redhat_access_pcm_ascension_common/pull/69/files?w=1`  
stratajs: `https://github.com/redhataccess/stratajs/pull/61/files?w=1`

**Comment by anujsi** [comment]
Reviewed.

File: app/cases/constants/strataCaseFields.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+export const strataCaseFields = [
```
Comment (vrathee) [inline app/cases/constants/strataCaseFields.js]: @praniredhat These are not strataCaseFileds, this is solrCaseFieldList

**Comment by praniredhat** [comment]
Retriggered build

File: app/cases/constants/strataCaseFields.js (hunk 1 above)
Comment (praniredhat) [reply app/cases/constants/strataCaseFields.js]: Oops that was a mistake

**Comment by praniredhat** [comment]
1) Renamed file strataCaseFields.js => solrCaseFields.js  
2) Renamed Var strataCaseFields => solrCaseFields  
Merging this

**Pull request merged**